packages/rich-text/src/component/index.js

```diff
@@ -26,7 +26,6 @@ export function useRichText( {
 	selectionEnd,
 	placeholder,
 	preserveWhiteSpace,
-	format = 'string',
```

Comment:
The block editor implementation of rich text wasn't passing a format prop at all.